### DISCUSSION TITLE
Default fcitx5 profile with settings

### DISCUSSION THREAD

**Discussion by hg2581** [fcitx/fcitx5#1109] [Q&A]
I have compiled fcitx5 myself under Rocky Linux 9, the only part missing as far as I can tell right now is the fcitx5-config-qt although I do have fcitx5-configuration which is supposed to launch the former.

However, in ~/.config/fctix5/profile I only have the the Group settings, nothing else. How is the default configuration profile with all its detailed settings generated and how do I do that?

I have the server running on my desktop but am so far unable to do anything with fctix5...

**Comment by wengxt** [comment]
Use flatpak instead of compiling yourself should be much easier.

I made a video about detailed steps in the past.
https://m.youtube.com/watch?v=Sn2ctL7PYKM

You do need fcitx5-config-qt if you need a gui to configure

**Reply by wengxt** [reply]
You might also miss the engine package.

Check https://fcitx-im.org/wiki/Input_method_engines to see a list of engines that fcitx supports.

**Comment by hg2581** [comment]
On July 31, 2024 11:55:21 PM EDT, Weng Xuetian ***@***.***> wrote:
>Use flatpak instead of compiling yourself should be much easier.
>
>I made a video about detailed steps in the past.
>https://m.youtube.com/watch?v=Sn2ctL7PYKM
>
>You do need fcitx5-config-qt if you need a gui to configure
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/fcitx/fcitx5/discussions/1109#discussioncomment-10207662
>You are receiving this because you authored the thread.
>
>Message ID:
>***@***.***>

No, I do not want flatpak.

Yes, my question was where I find the code for fcitx5-config-qt? Does it have its own GitHub repository? Could you please post the location?

**Comment by hg2581** [comment]
On July 31, 2024 11:57:10 PM EDT, Weng Xuetian ***@***.***> wrote:
>You might also miss the engine package.
>
>Check https://fcitx-im.org/wiki/Input_method_engines to see a list of
>engines that fcitx supports.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/fcitx/fcitx5/discussions/1109#discussioncomment-10207673
>You are receiving this because you authored the thread.
>
>Message ID:
>***@***.***>

I have already compiled fcitx5-chinese-addon and fcitx5-table-extra.

**Comment by hg2581** [comment]
Very happy to report that I got everything working! I had misunderstood how to configure fcitx-configuretool. I have, however, yet to compile either fcitx-libpinyin or fcitx-googlepinyin which both seem to be for fcitx4, ie. the previous version. They are also not mentioned on the page with information on compiling fcitx5 or its various libraries.

Am I correct in assuming that neither fcitx-libpinyin or fcitx-googlepinyin are needed for fcitx5 but were for fcitx4?

I have looked around in the myriad settings in fcitx5-config-qt but did not find a setting for the size of the pinyin characters in the input module? I know I could change that size in an older version of fcitx5 but have I missed it now or do I change this size somewhere else?

I only did some very quick testing in a couple of applications and it seemed to work but need to do more extensive testing. By the way, I am using X11, not Wayland, for my Rocky Linux 9 installation.

**Reply by wengxt** [reply]
https://www.csslayer.info/wordpress/fcitx-dev/a-new-pinyin-input-method-that-might-be-slightly-better-than-sunpinyin/ 
This blog has a brief introduction about the new pinyin (in fcitx5-chinese-addons)

If you are asking for changing font size, check the classicui addon configuration with configtool

**Comment by hg2581** [comment]
Good, I will then remove both fcitx-libpinyin and fcitx-googlepinyin from my bash script compiling fcitx5.

Found where to set the font size for the input method. Thank you.
